spec/spec_helper.rb

```diff
@@ -22,6 +26,24 @@
 ENV['RACK_ENV'] ||= 'test'
 ARGV = []
 
+## BrowserStack config
+
```

Comment:
@jackdwalker  This doesn't seem ideal but if it works it can be picked up at some point later

Reply:
To be clear, i am referring to the monkey patching. i think its just worth noting and possibly making a plan to action it

Reply:
@jcrew99 - that is not my code. That is in the existing BeEF master. Feel free to open an issue on it though.

Reply:
ahaha cool